killerbee/GoodFETCCSPI.py

```diff
@@ -198,6 +197,20 @@ def pokeram(self,adr,data):
         return;
     
     lastpacket=range(0,0xff);
+
+    def RF_txrxpacket(self,packet,timeout=1):
```

Comment:
Using this for zbstumbler type use cases may be helpful to get a potentially quicker RX to a forged beacon request? CC @taylorcenters